data/json/recipes/basecamps/recipe_modular_saltworks/recipe_modular_saltworks_log.json

```diff
@@ -0,0 +1,92 @@
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_saltpan_log_northeast",
+    "blueprint_name": "northeast saltpan",
+    "blueprint_requires": [ { "id": "fbmsw_0" } ],
+    "blueprint_provides": [ { "id": "fbmsw_northeast" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_northeast" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_shack_log_southeast",
+    "description": "We could use a storage room for chemicals and tools, so build a log shack with a roof on the southeast side of the saltworks field.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_shack_log_southeast",
+    "blueprint_name": "southeast storage room",
+    "blueprint_requires": [ { "id": "fbmsw_0" } ],
+    "blueprint_provides": [ { "id": "fbmsw_southeast" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_southeast" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_brewery0_log_southwest",
+    "description": "We could take advantage of the the designation of this area to build a log brewery.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_brewery0_log_southwest",
+    "blueprint_name": "southwest brewery",
+    "blueprint_requires": [ { "id": "faction_base_saltworks_0" } ],
+    "blueprint_provides": [ { "id": "fbmsw_southwest" }, { "id": "brewery" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_southwest" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_brewery1_log_southwest",
+    "description": "We could uprade the workspace in the brewery to add a still, wood stove and work table.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_brewery1_log_southwest",
+    "blueprint_name": "southwestwest brewery1",
+    "blueprint_requires": [ { "id": "fbmsw_southwest" } ],
+    "blueprint_provides": [ { "id": "fbmsw_southwest1" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_southwest1" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_brewer2_log_southwest",
+    "description": "We could use fill out the space with extra kegs and storage racks.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_brewery2_log_southwest",
+    "blueprint_name": "southwest brewery2",
+    "blueprint_requires": [ { "id": "fbmsw_southwest1" } ],
+    "blueprint_provides": [ { "id": "fbmsw_southwest2" } ],
```

Comment:
excludes id is still misspelled. should be identical to requires.

Reply:
I followed the same format from the evac shelter recipe? I think?
``    "blueprint_requires": [ { "id": "fbmc_shelter_bedroom1" } ],
    "blueprint_provides": [ { "id": "fbmc_shelter_bedroom2" } ],
    "blueprint_excludes": [ { "id": "fbmc_shelter_bedroom2" } ],``

Reply:
Yeah that's fine, just check the spelling of southwest,  3 are souththwest

Reply:
Ah! It's like that thing where as long as the first and last letter are right the brain just fills in the rest.  Thanks clarifying neither I or my wife could find the spelling error.

Reply:
Yeah, it's hard to spot, the eye goes right past it : )